lib/dialects/mysql/query.js

```diff
@@ -92,7 +92,22 @@ class Query extends AbstractQuery {
       this.handleInsertQuery(data);
 
       if (!this.instance) {
-        result = data[this.getInsertIdField()];
+        // handle bulkCreate AI primiary key 
+        if (
+          data.constructor.name === 'ResultSetHeader'
+          && this.model
+          && this.model.autoIncrementField
+          && this.model.autoIncrementField === this.model.primaryKeyAttribute
+          && this.model.rawAttributes[this.model.primaryKeyAttribute]
+        ) {
+          const startId = data[this.getInsertIdField()];
+          result = [];
+          for (let i = startId; i < startId + data.affectedRows; i++) {
```

Comment:
Is manually calculating the ids after `last_insert_id ` the generally acceptable way of figuring out multi insert ids?
If so i'd like some internal references. This solution also fails to take into account that the increment interval can be different from 1.